api/serializers.py

```diff
@@ -62,9 +73,14 @@ class Meta:
         model = LearningModule
         fields = '__all__'
 
+    def create(self, validated_data):
+        learning_unit = validated_data.pop("learning_unit")
+        new_module = LearningModule(**validated_data)
```

Comment:
Should this be:

new_module = LearningModule(learning_unit)

Otherwise, the first element is always missed out.